web/src/components/CityContentLayout.tsx

```diff
@@ -63,15 +64,21 @@ const LocationLayout = (props: PropsType): ReactElement => {
       disableScrollingSafari={disableScrollingSafari}
       fullWidth={fullWidth}
       header={
-        <LocationHeader
+        <CityContentHeader
           cityModel={cityModel}
           languageChangePaths={languageChangePaths}
           viewportSmall={viewportSmall}
           languageCode={languageCode}
           route={route}
+          isSidebarOpen={isSidebarOpen}
+          setIsSidebarOpen={setIsSidebarOpen}
         />
       }
-      footer={!isLoading && showFooter ? <LocationFooter city={cityModel.code} language={languageCode} /> : null}
+      footer={
+        !isLoading && showFooter && !isSidebarOpen ? (
```

Comment:
I think this should not be dependend on whether the sidebar is currently opened or closed. Imo there should not be a footer shown here at all if we are on a small viewport and have the sidebar.

Reply:
Hmmm, I feel like the imprint and privacy should always be easily seen and available, not just after clicking on an icon that you first have to scroll up to find, that's not very intuitive to me.

Reply:
Did not really think about that but according to the issue it should be moved and therefore not displayed here for city content. Therefore I'd propose to check back with the rest of the team, especially UI/UX to discuss what to do. I am personally fine with both. But I think there was a reason to move it to the sidebar in the first place.

Reply:
Sorry, forgot to comment, yes, the footer is now not shown on the main screen on small screens, it's only in the sidebar there.